ZCS-18232 : Remove zip and zipstore options from zimbraBackupDedupeCompressionType LDAP attribute.

Gopal-Moolchandani opened the pull request.

Gopal-Moolchandani committed:
ZCS-18008 : Backup: Capture new full backup parameters in session.xml

Added logic for Callback

ZCS-18008 : Changed the since version to 10.1.14

ZCS-18232 : Remove zip and zipstore options from zimbraBackupDedupeCompressionType LDAP attribute.

ZCS-18232 : Added ZimbraBackupCrossSessionDedupeEnabled attribute in LDAP attributes.

Refactored according to modifications in requirement

Modified the since version of new Backup LDAP attributes from 10.1.14 to 10.1.15

Changed the default value of ZimbraBackupCrossSessionDedupeEnabled to false

Added the support for csd common backup

Refactored

Refactored the attribute name

ZCS-18377 : Handle restore from CSD backups where CSD common backup is compressed and CSD delta backup is not compressed (or vice-versa)

Modified the logic to remove cyclic dependency

ZCS-18461: Unable to save or enabled dedupe and CSD backup together from admin console

ZCS-18469 : Fixed backup creation with nocsd flag by adding it as a request parameter

preshitaagrawal committed:
Fixed duplicate instance modification received from device

umagmrit committed:
Revert "ZCS-17622:Apache mina version update (#1745)"

This reverts commit 2384fdf06ff5d9cb84b1530e79ae35ecce7fa0e8.

k-kato commented on store/conf/attrs/zimbra-attrs.xml:
```diff
@@ -10613,4 +10613,25 @@ TODO: delete them permanently from here
+  </desc>
+</attr>
+
+<attr id="4149" name="ZimbraBackupCrossSessionDedupeEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="10.1.15">
```
use lower case `z` in `zimbra`
```suggestion
<attr id="4149" name="zimbraBackupCrossSessionDedupeEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="10.1.15">
```

chetanlanje74 committed:
ZCS-18299:implementation & Enhancement for Code-coverage in zimbra-unit-tests pipeline (#1773)

* Update build-common.xml

* Update build-common.xml

* ZCS-18299:Adding jacoco code-coverage for zm-mailbox/store,common,client,soap components

* ZCS-18299:adding skipcoverage condition for coverage target

shrutig0510 committed:
ZBUG-3131 Remove hard-coded password from createBulkAccounts (#1666)

ashishkataria86 committed:
ZBUG-4288 (#1664)

viditZim committed:
ZBUG-4197: Proxy servlet filtering (#1775)

Co-authored-by: Ron Straight <ron.straight@synacor.com>

umagmrit committed:
ZCS-18242 : add target for sonar scan

viditZim committed:
ZBUG-3974 ZBUG-3974 Standard name and Daylight name mapped according to windows clients (#1774)

ZBUG-3974 updated license block

ZBUG-3974 added logic to create timezone map dynamically from windows timezone information files

ZBUG-3974 Modified file name

umagmrit committed:
ZCS-18242 : Fix typo in SonarQube build echo
